setup.cfg

```diff
@@ -55,6 +55,7 @@ install_requires =
     click
     click-plugins
     bel_resources>=0.0.3
+    more_itertools
```

Comment:
didnt know this

Reply:
yeah i wish I had started using this a long time ago...